packages/ui-components/src/Input/EmailPassword.module.scss

```diff
@@ -6,9 +6,7 @@
 }
 
 .new-password-validation-container {
-  display: flex;
-  flex-flow: column wrap;
-  list-style: none;
+  column-count: 2;
```

Comment:
instead of implementing with column wrap, which will create as many rows as possible before wrapping to the next column, we can enforce a max column number, behaviour is shown in PR description screenshot. Max # of columns confirmed by design on ticket: https://cockroachlabs.atlassian.net/browse/DS-13